R/module_teal.R

```diff
@@ -114,12 +97,7 @@ ui_teal <- function(id, modules) {
     ui_snapshot_manager_panel(ns("snapshot_manager_panel")),
     ui_filter_manager_panel(ns("filter_manager_panel"))
   )
-
-  navbar <- .teal_navbar(
-    id = ns("active_module_id"),
-    nav_items = nav_elements,
-    tab_content = module_items$tab_content
-  )
+  navbar <- .teal_navbar_append(navbar, nav_elements)
```

Comment:
@vedhav this is the most important change in this commit. `navbar` is created in `ui_teal_module` instead of here and child is appended instead.